### DISCUSSION TITLE
Migrate/Share data from old cluster to new cluster

### DISCUSSION THREAD

**Discussion by SunSparc** [grafana/mimir#6144] [Help and support] [answered]
I have a Mimir cluster that has been running for around six months. It works very well. Recently I was asked to move the cluster to a new datacenter. I deployed a new cluster using the same configurations, using GCS for remote storage. The new cluster is working well also, however, the historical data from the previous cluster does not show up when using Grafana charts to view the data in Mimir.

Looking in the GCS bucket, I still see all the data from the previous cluster. Is there something I need to do in order for the new cluster to pick up the old data?

**Comment by SunSparc** [comment, marked as answer]
Time. Time was the answer. =) I just had to wait. Apparently the new cluster had to process all the historical data from the previous cluster. Now my charts in the new cluster are showing the same stuff as the old.

**Reply by dimitarvdimitrov** [reply]
out of curiosity, how long did you have to wait for the data to be discovered by the new cluster?

**Reply by SunSparc** [reply]
I finished the migration around the end of a workday. When I checked again the next morning the data was there. Which seems to indicate that it was less than 12 hours.
